### PR TITLE
v1.7.2

### PR DESCRIPTION
**Pull request by derekparker**
Thanks to everyone who contributed to this release!

@aarzilli @polinasok @suzmue @krobelus @hyangah @gabriel-vasile @cmol

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,31 @@
...
+* Print config output strings quouted (@aarzilli, @krobelus)
+* Update check for system goroutines (@suzmue)
+* DAP: Halt before detach in Stop (@polinasok)
+* DAP: Do not send halt request if debuggee is not running (@suzmue)
+
+### Changed
+
+* Include selected goroutine in threads request (@suzmue)
+* Remove individual OS install instructions (@gabriel-vasile)
+* DAP: Show decimal / hex values for uint
```
Comment (polinasok) [inline CHANGELOG.md:29]: missing (@suzmue)

**Review by polinasok — COMMENTED** [review]
@hyangah Do we need to get the fix for the build error pop-up into this release?

**Comment by polinasok** [comment]
> @hyangah Do we need to get the fix for the build error pop-up into this release?

@hyangah How about the version warning since we are turning off the flag in vscode (#2684)

File: CHANGELOG.md (hunk 1 above)
Comment (derekparker) [reply CHANGELOG.md]: 🤦‍♂️ Fixed.

**Comment by hyangah** [comment]
> > @hyangah Do we need to get the fix for the build error pop-up into this release?
> 
> @hyangah How about the version warning since we are turning off the flag in vscode (#2684)

It would be nice to have a warning, but it is not a blocker.

**Review by aarzilli — APPROVED** [review]

**Comment by polinasok** [comment]
> > > @hyangah Do we need to get the fix for the build error pop-up into this release?
> > 
> > @hyangah How about the version warning since we are turning off the flag in vscode (#2684)
> 
> It would be nice to have a warning, but it is not a blocker.

@derekparker, the PRs for these fixes are already pending and approved by @aarzilli: #2671 + #2710, #2684. How much trouble would it be to merge them in? These are not critical breakages, but are usability pain points that users have explicitly surfaced. Sorry, I did not flag this sooner. I did not realize you were preparing for another monthly release. I will pay more attention in the future. I will let you use your judgement on whether to get them into this release or delay until the next. Thank you.

**Comment by derekparker** [comment]
> > > > @hyangah Do we need to get the fix for the build error pop-up into this release?
> > > 
> > > 
> > > @hyangah How about the version warning since we are turning off the flag in vscode (#2684)
> > 
> > 
> > It would be nice to have a warning, but it is not a blocker.
> 
> @derekparker, the PRs for these fixes are already pending and approved by @aarzilli: #2671 + #2710, #2684. How much trouble would it be to merge them in? These are not critical breakages, but are usability pain points that users have explicitly surfaced. Sorry, I did not flag this sooner. I did not realize you were preparing for another monthly release. I will pay more attention in the future. I will let you use your judgement on whether to get them into this release or delay until the next. Thank you.

As of now I'm leaning towards merging them in for the 1.7.3 release as there's nothing there that seems critical. The 1.7.3 release will go out a month from now, would that be a good enough timeline for the above patches or is there more urgency to have them released today / tomorrow with 1.7.2?

**Review by polinasok — APPROVED** [review]

**Pull request merged**